client/my-sites/plans/jetpack-plans/product-store/hooks/use-pricing-breakdown.tsx

```diff
@@ -18,7 +19,7 @@ export const usePricingBreakdown = ( {
 		product?.monthlyProductSlug || ''
 	);
 
-	const bundlePrice = discountedPrice || originalPrice || 0;
+	const bundlePrice = isNumber( discountedPrice ) ? discountedPrice : originalPrice || 0;
```

Comment:
Do you remember what didn't work without this change?

Reply:
Same as the places:
> That's needed because discountedPrice would be 0 for Advanced plan now, which is falsy, so we rather have to check if it's a number to have the wanted behaviour